templates/default/fulldoc/html/full_list.erb

```diff
@@ -26,7 +26,10 @@
           <% end %>
         </div>
 
-        <div id="search">Search: <input type="text" /></div>
+        <div id="search">
+          <label for="search-class">Search:</label>
+          <input id="search-class" type="text" />
+        </div>
```

Comment:
This is just adding a label for search for screen readers to know what it's for.